src/QMCWaveFunctions/Jastrow/RadialJastrowBuilder.cpp

```diff
@@ -359,6 +366,10 @@ bool RadialJastrowBuilder::createJ1(xmlNodePtr cur)
       {
 	functor->cutoff_radius = 10.0;
       }
+      if (cusp > 0) 
```

Comment:
Is it better if we set 0 as default.
Here check if <0, abort with error. Then else always call setCusp.

Reply:
Change my mind a bit. Put 0 as default and allow negative cusp for protons.